Chutong data

Chutong2000 opened the pull request:
ready to be merged

Chutong2000 committed:
My message

Chutong2000 committed:
My message

Chutong2000 committed:
My message

Chutong2000 committed:
add describe

Chutong2000 committed:
My message

Chutong2000 committed:
My message

Chutong2000 committed:
My message

Chutong2000 committed:
My message

Chutong2000 committed:
My message

Chutong2000 committed:
My message

Chutong2000 committed:
My message

Chutong2000 committed:
My message

Chutong2000 committed:
My message

Chutong2000 committed:
My message

Chutong2000 committed:
My message

Chutong2000 committed:
My message

Chutong2000 committed:
My message

Chutong2000 committed:
resolve comfilct

Merge branch 'main' of https://github.com/TLdgh/Categorical_Assign1 into chutong_data

# Conflicts:
#	.gitignore
#	Categorical_Assign1.Rmd

TLdgh reviewed (approved):
Approve Chutong's merge request.

Chutong2000 committed:
change the box plot

Chutong2000 committed:
change the order

Chutong2000 committed:
little change on size

Chutong2000 committed:
html change

Chutong2000 committed:
delete html

Merge branch 'main' of https://github.com/TLdgh/Categorical_Assign1 into chutong_data

# Conflicts:
#	Categorical_Assign1.html

TLdgh committed:
minor fixes

TLdgh reviewed (approved):
Branch approved.
